building/build-debs/homeworld-keysystem/config/keyserver.yaml

```diff
@@ -0,0 +1,268 @@
+    key: etcd-server.key
+    cert: etcd-server.pem
+
+  etcd-client:
+    type: TLS
+    key: etcd-client.key
+    cert: etcd-client.pem
+
+  kubernetes:
+    type: TLS
+    key: kubernetes.key
+    cert: kubernetes.pem
+
+  serviceaccount:
+    type: static
+    key: serviceaccount.key
+    cert: serviceaccount.pem
+
+accounts:
+  - principal: eggs-benedict.mit.edu
+    group: master-nodes
+    limit-ip: true
+    metadata:
+      ip: 18.181.0.97
+      hostname: eggs-benedict
+      schedule: false
+
+  - principal: huevos-rancheros.mit.edu
+    group: master-nodes
+    limit-ip: true
+    metadata:
+      ip: 18.181.0.104
+      hostname: huevos-rancheros
+      schedule: false
+
+  - principal: ole-miss.mit.edu
+    group: master-nodes
+    limit-ip: true
+    metadata:
+      ip: 18.181.0.106
+      hostname: ole-miss
+      schedule: false
+
+  - principal: grilled-cheese.mit.edu
+    group: worker-nodes
+    limit-ip: true
+    metadata:
+      ip: 18.181.0.128
+      hostname: grilled-cheese
+      schedule: true
+
+  - principal: avocado-burger.mit.edu
+    group: worker-nodes
+    limit-ip: true
+    metadata:
+      ip: 18.181.0.129
+      hostname: avocado-burger
+      schedule: true
+
+  - principal: french-toast.mit.edu
+    group: worker-nodes
+    limit-ip: true
+    metadata:
+      ip: 18.181.2.232
+      hostname: french-toast
+      schedule: true
+
+  - principal: egg-sandwich.mit.edu
+    group: supervisor-nodes
+    limit-ip: true
+    metadata:
+      ip: 18.181.0.253
+      hostname: egg-sandwich
+      schedule: false
+
+  - principal: cela@ATHENA.MIT.EDU
+    disable-direct-auth: true
+    group: root-admins
+
+  - principal: dukhovni@ATHENA.MIT.EDU
+    disable-direct-auth: true
+    group: root-admins
+
+  - principal: ikdc@ATHENA.MIT.EDU
+    disable-direct-auth: true
+    group: root-admins
+
+groups:
+  root-admins:
+  nodes:
+  supervisor-nodes:
+    subgroupof: nodes
+  kubernetes-nodes:
+    subgroupof: nodes
+  worker-nodes:
+    subgroupof: kubernetes-nodes
+  master-nodes:
+    subgroupof: master-nodes
+
+grants:
+
+  # ADMIN ACCESS TO THE RUNNING CLUSTER
+
+  access-ssh:
+    group: root-admins
+    privilege: sign-ssh
+    authority: ssh-user
+    lifespan: 4h
+    ishost: false
+    common-name: temporary-ssh-grant-(principal)
+    allowed-names:
+    - root
+
+  access-etcd:
+    group: root-admins
+    privilege: sign-tls
+    authority: etcd-client
+    lifespan: 4h
+    ishost: false
+    common-name: temporary-etcd-grant-(principal)
+
+  access-kubernetes:
+    group: root-admins
+    privilege: sign-tls
+    authority: kubernetes
+    lifespan: 4h
+    ishost: false
+    common-name: temporary-kube-grant-(principal)
+
+  # MEMBERSHIP IN THE CLUSTER
+
+  bootstrap:
+    group: root-admins
+    privilege: bootstrap-account
+    scope: nodes
+    lifespan: 1h
+
+  renew-keygrant:
+    group: nodes
+    privilege: sign-tls
+    authority: keygranting
+    lifespan: 960h  # fourty day lifespan
```

Comment:
Since this one is pretty minor, and has already been fixed in the future commit d81ebe69aca170a9ef87d1ee76b77322af6a3d6d on feature-admin-tools, I'm skipping dealing with it now.